website/content/en/docs/faq.md

```diff
@@ -0,0 +1,122 @@
+
+### How does Karpenter interact with AWS node group features?
+Provisioners are designed to work alongside static capacity management solutions like EKS Managed Node Groups and EC2 Auto Scaling Groups.
+You can manage all capacity using provisioners, use a mixed model with dynamic and statically managed capacity, or use a fully static approach.
+We expect most users will use a mixed approach in the near term and provisioner-managed in the long term.
+
+
+### How does Karpenter interact with Kubernetes features?
+* Kubernetes Cluster Autoscaler: Karpenter can work alongside cluster autoscaler.
+See [Kubernetes cluster autoscaler]({{< ref "/docs/concepts/#kubernetes-cluster-autoscaler" >}}) for details.
+* Kubernetes Scheduler: Karpenter focuses on scheduling pods that the Kubernetes scheduler has marked as unschedulable.
+See [Scheduling]({{< ref "/docs/concepts/#scheduling" >}}) for details on how Karpenter interacts with the Kubernetes scheduler.
+
+## Provisioning
+### What features does the Karpenter provisioner support?
+See [Provisioner API]({{< ref "/docs/provisioner" >}}) for provisioner examples and descriptions of features.
+
+### Can I create multiple (team-based) provisioners on a cluster?
+Yes, provisioners can identify multiple teams based on labels.
+See [Provisioner API]({{< ref "/docs/provisioner" >}}) for details.
+
+### If multiple provisioners are defined, which will my pod use?
+
+By default, pods will use the rules defined by a provisioner named default.
+This is analogous to the default scheduler.
+To select an alternative provisioner, use the node selector `karpenter.sh/provisioner-name: alternative-provisioner`.
+You must either define a default provisioner or explicitly specify `karpenter.sh/provisioner-name node selector`.
+
+
+### Can I set total limits of CPU and memory for a provisioner?
+!!! NEEDS INFO !!!
+
+### Can I mix spot and on-demand EC2 run types?
+Yes, see [Example Provisioner Resource]({{< ref "/docs/provisioner/#example-provisioner-resource" >}}) for an example.
+
+### Can I restrict EC2 instance types?
+
+* Attribute-based requests are currently not possible.
+* You can select instances with special hardware, such as gpu.
+
+### Can I identify EC2 instances using common labels?
+!!! NEEDS INFO !!!
+
+## Workloads
+
+### How can someone deploying pods take advantage of Karpenter?
+
+See [Application developer]({{< ref "/docs/concepts/#application-developer" >}}) for descriptions of how Karpenter matches nodes with pod requests.
+
+### How do I use Karpenter with the AWS load balancer controller?
+
+* Set the [ALB target type]({{< ref "https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/guide/ingress/annotations/#target-type" >}}) to IP mode for the pods 
+* Set [readiness gate]({{< ref "https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/deploy/pod_readiness_gate/" >}}) on the namespace.
+The default is round robin at node level.
```

Comment:
Doesn't this depend on 1/ the TargetType and 2/ the traffic policy (cluster or local)?